releases/client/3.10/settings

```diff
@@ -1 +1,2 @@
-OSES="el7 el8 el9"
+FULLVERSION='3.10.0-rc2'
```

Comment:
Client doesn't need that

Reply:
'twas an artifact of branching, removing